Sketch seed (#26050)
matplotlib/matplotlib: doc/users/next_whats_new/sketch_seed.rst

``sketch_seed`` parameter for rcParams

> ----------------------------------------------------

Reviewer: Should be same length as the title.

`~matplotlib.rcParams` now has a new parameter ``path.sketch_seed``.

> It's default value is 0 and accepted values are any non negative integer.

Reviewer: ```suggestion
Its default value is 0 and accepted values are any non negative integer.
```

> This allows the user to set a value to the seed for the internal Pseudo number generator in one of three ways.

Reviewer: ```suggestion
This allows the user to set a value to the seed for the internal pseudo random number generator in one of three ways.
```

1) Directly changing the rcParam:

> Eg. rcParams['path.sketch_seed']= 20

Reviewer: ```suggestion

    rcParams['path.sketch_seed'] = 20
```

2) Passing a value to the new *seed* parameter of `~matplotlib.pyplot.xkcd` function:

> Eg. plt.xkcd(seed=20)

Reviewer: ```suggestion

    plt.xkcd(seed=20)
```

3) Passing a value to the new *seed* parameter of `~matplotlib.artist.Artist.set_sketch_params` function:

> Eg. Artist.set_sketch_params(seed=20)

Reviewer: ```suggestion

    Artist.set_sketch_params(seed=20)
```

> Note that using any one of these 3 methods changes the value of rcParams['path.sketch_seed'] variable.

Reviewer: ```suggestion
Note that using any of these three methods changes the value of :rc:`path.sketch_seed`.
```

> The seed will also have a rolling(auto incrementing) characteristic.

Reviewer: ```suggestion
The seed will also have a rolling (auto incrementing) characteristic.
```
